worker/src/main/java/org/apache/celeborn/service/deploy/worker/congestcontrol/TimeSlidingHub.java

```diff
@@ -51,20 +51,20 @@ public interface TimeSlidingNode extends Cloneable {
   }
 
   // 1 second.
-  private final int intervalPerBucketInMills = 1000;
+  protected final int intervalPerBucketInMills = 1000;
   private final int maxQueueSize;
-  private N sumNode;
+  private Pair<N, Integer> sumInfo;
 
   private final LinkedBlockingDeque<Pair<Long, N>> _deque;
 
   public TimeSlidingHub(int timeWindowsInSecs) {
     this._deque = new LinkedBlockingDeque<>();
     this.maxQueueSize = timeWindowsInSecs * 1000 / intervalPerBucketInMills;
-    this.sumNode = newEmptyNode();
+    this.sumInfo = Pair.of(newEmptyNode(), 0);
   }
 
-  public N sum() {
-    return sumNode;
+  public Pair<N, Integer> sum() {
```

Comment:
Why we need to change this? How abt simply return 0 if `getCurrentTimeWindowsInMills` is 0 in `avgBytesPerSec`? 

I'm thinking if we want to change the value of `intervalPerBucketInMills` to be smaller in future, exposing the index of node could introduce difficulities(we cannot easily to calculate `getCurrentTimeWindowsInMills`)

Reply:
This change is also to make the returned avg value more accurate, so that numBytes and slice size are consistent